### DISCUSSION TITLE
Code snippets: Prisma language support

### DISCUSSION THREAD

**Discussion by steebchen** [GitbookIO/community#77] [Feature Requests]
### What's your feature/idea?

It would be great if code snippets could support the [Prisma](https://www.prisma.io) language. It's suppored by GitHub as well via [linguist](https://github.com/github/linguist/pull/4668/files).

### What's your desired outcome?

Provide documentation for custom Prisma clients, such as the Go client.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by SamyPesse** [comment]
We are using `prismjs` for syntax highlighting, it looks it like it doesn't support Prisma at the moment (https://github.com/PrismJS/prism/issues/3462), but it should be feasible by taking the custom language definition implemented by the tea at Prisma for their docs.

**Reply by SamyPesse** [reply]
I've opened a pull request internally, and we'll ship support for the Prisma syntax early next week :)

**Reply by steebchen** [reply]
Awesome, didn't expect such a swift response, thank you!

**Reply by SamyPesse** [reply]
You're welcome :) It shipped today; you might need to hard-refresh your browser for it to work.

**Reply by steebchen** [reply]
Thanks, doesn't seem to be working yet tho:

https://steebchen.gitbook.io/prisma-client-go/quickstart

https://github.com/steebchen/prisma-client-go-2/blob/main/docs/quickstart.md

<img width="946" alt="image" src="https://user-images.githubusercontent.com/5013932/217148978-c1384737-6122-443d-af0b-64f459d6ca62.png">

**Reply by SamyPesse** [reply]
You need to update the page by making a commit from GitHub; it'll trigger an import in GitBook and update the content there.

**Reply by steebchen** [reply]
ah right, it does work now, thx!
